aws/resource_aws_iam_group_policy.go

```diff
@@ -131,8 +148,13 @@ func resourceAwsIamGroupPolicyDelete(d *schema.ResourceData, meta interface{}) e
 	return nil
 }
 
-func resourceAwsIamGroupPolicyParseId(id string) (groupName, policyName string) {
+func resourceAwsIamGroupPolicyParseId(id string) (groupName, policyName string, err error) {
 	parts := strings.SplitN(id, ":", 2)
+	if len(parts) != 2 {
```

Comment:
This check should also include empty string checks for before and after the colon 😄 

```suggestion
	if len(parts) != 2 || parts[0] == "" || parts[1] == "" {
```

Reply:
Done.